source/change-streams/tests/unified/change-streams-pre_and_post_images.yml

```diff
@@ -0,0 +1,350 @@
+description: "change-streams-pre_and_post_images"
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "6.0.0"
+    topologies: [ replicaset, sharded-replicaset, load-balanced ]
```

Comment:
I believe the CI is failing because of this line... Not sure why, though.

Reply:
Looking into this now. I just realized change-streams were never added to the Unified Test Format Makefile and just did so now. It validates locally, so I'll have to sort out how the CI task differs.